bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2B_ggfusion_b-associatedproduction/run2Template_cards/template_HToZATo2L2B_200_50_1_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8/HToZATo2L2B_200_50_1_bbH4F_TuneCP5_13TeV-amcatnlo_pythia8_customizecards.dat

```diff
@@ -0,0 +1,18 @@
+# Settings for the Higgs potential 
+set param_card higgs 1 0.0655307646
+set param_card higgs 2 0.0655307646
+set param_card higgs 3 0.161002788
+# Settings 2hdm parameters 
+set param_card frblock 1 1.0     # tanbeta 
+set param_card frblock 2 0.99    # sinbma 
+# Settings for the scalar bosons masses 
+set param_card mass 25 125.0     # h1: h
+set param_card mass 35 200.0     # h2: H
+set param_card mass 36 50.0      # h3: A
+set param_card mass 37 200.0     # hc: Hc
```

Comment:
what about the width of 37? its mass is changing.

Reply:
This card in the `run2Template_cards/ ` dir . I added it to make the script I uploaded a working version if anyone wants to try. 
I no longer using the `customized_cards.dat.`

In the `example_cards/`  which represents an example of the gridpack cards that are going to be submitted for the official production , you can see that the full table of BR and decay width , masses, free-parameters of the 2hdm model etc... are added to the `param_card.dat` for pdgid (35, 36, 37 and 23 ).